### PR TITLE
Update issues in management binary commander docker and source pages in core docs 2.1.5 - closes #599

### LINKED ISSUE #599: Update issues in management, binary, commander, docker & source pages in Core docs 2.1.5
Management. Lisk core, small c should be large C
 Binary commands: Source Administration page doesn’t exist
 Docker commands: URLs are written out
 Source commands: Mainnet/Testnet
 Handle config files: Docker Administration page doesn’t exist / Sometimes its Commander, sometimes Lisk Commander in the selection box

Parent issue #591

### PR DESCRIPTION
**Pull request by chris529**
closes #599

### CODE REVIEW

**Review by Tschakki — CHANGES_REQUESTED** [review]
please fix the conflicts first.

File: modules/ROOT/pages/management/commander.adoc [hunk 1]
```diff
@@ -65,7 +65,7 @@ lisk core:uninstall lisk-testnet
 
 === Status
 
-Check the status of the Lisk Core node.
+Check the status of the Lisk Core Node.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/commander.adoc]:
```suggestion
Check the status of the Lisk Core node.
```

File: modules/ROOT/pages/management/docker.adoc [hunk 2]
```diff
@@ -6,12 +6,11 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 :page-next-title: Configuration
 :page-previous-title: Docker installation
-
+:url_docker_compose_ports: https://docs.docker.com/compose/reference/port/
+:url_docker_compose_compose: https://docs.docker.com/compose/compose-file/#environment
 :url_docker_compose_exec: https://docs.docker.com/compose/reference/exec/
 :url_lisk_snapshots: http://snapshots.lisk.io
-
 :url_config: management/configuration.adoc
-:url_config_cl: reference/config.adoc#clo
```
Comment (Tschakki) [inline modules/ROOT/pages/management/docker.adoc:14]: Why remove the link attribute? It is still used in the document.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/management/docker.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/management/docker.adoc:14]: OK, I have replaced it.. :-)

Comment (Tschakki) [reply modules/ROOT/pages/management/docker.adoc:14]: I think you replaced it in the wrong file (mannagement/commander.adoc), please fix.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/management/docker.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/management/docker.adoc:14]: OK, I have put the link attribute back in management/docker.adoc

File: modules/ROOT/pages/management/docker.adoc [hunk 3]
```diff
@@ -6,11 +6,10 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-previous: /lisk-core/setup/docker.html
 :page-next-title: Configuration
 :page-previous-title: Docker installation
-
+:url_docker_compose_ports: https://docs.docker.com/compose/reference/port/
+:url_docker_compose_compose: https://docs.docker.com/compose/compose-file/#environment
 :url_docker_compose_exec: https://docs.docker.com/compose/reference/exec/
 :url_lisk_snapshots: http://snapshots.lisk.io
-
-:url_config: management/configuration.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/management/docker.adoc:13]: Why did you remove this link? It is used in this document. Please restore.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/management/docker.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/management/docker.adoc:13]: Ah yes, sorry that was an error...It has now been restored :-)

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**